src/configuration.js

```diff
@@ -155,16 +152,18 @@ export default class Configuration {
      *        as process-level tags on the Tracer itself.
      */
     static initTracer(config, options = {}) {
-        let reporters = [];
         let reporter;
         let sampler;
+        if (options.metrics) {
+            options.metrics = new Metrics(options.metrics);
```

Comment:
hypothetically they can pass in whatever they want given it's javascript. We do have a comment above that says "metrics factory"